tests/integration/bootstrap.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+use Ibexa\Contracts\Core\Test\Persistence\Fixture\FixtureImporter;
+use Ibexa\Tests\Core\Repository\LegacySchemaImporter;
+use Ibexa\Tests\Integration\FieldTypeRichText\IbexaTestKernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+
+require_once dirname(__DIR__, 2) . '/vendor/autoload.php';
+
+$kernel = new IbexaTestKernel('test', true);
+$kernel->boot();
+
+$application = new Application($kernel);
+$application->setAutoExit(false);
+
+if (getenv('DATABASE_URL') !== false && 'sqlite' !== substr(getenv('DATABASE_URL'), 0, 6)) {
```

Comment:
This can be simplified as:
```suggestion
$databaseUrl = getenv('DATABASE_URL');
if ($databaseUrl !== false && strpos($databaseUrl, 'sqlite') !== 0) {
```
cc @Steveb-p this is why I don't like boilerplate like this. We find something to improve and now should update all packages using it. Is there a possibility to somehow wrap this boilerplate to be used from within one common package? ATM core, but in the future maybe dedicated test package.